src/adafruit_blinka/microcontroller/mcp2221/mcp2221.py

```diff
@@ -396,7 +474,7 @@ def dac_write(self, pin, value):
         report[4] = 1 << 7 | (value & 0b11111)
         self._hid_xfer(report)
 
-    # pylint: enable=unused-argument
+   # pylint: enable=unused-argument
```

Comment:
```suggestion
    # pylint: enable=unused-argument
```
I accidentally lost a space while restoring this line :)